File: attribouter/src/main/java/me/jfenn/attribouter/data/github/UserData.java
package me.jfenn.attribouter.data.github;

public class UserData extends GitHubData {

    private String login;
    private String name;
    private String avatar_url;
    private String html_url;
    private String blog;
    private String email;
    private String bio;

    public UserData(String login) {
        super("https://api.github.com/users/" + login);
    }

}

File: attribouter/src/main/java/me/jfenn/attribouter/provider/net/data/UserData.kt
package me.jfenn.attribouter.provider.net.data

import me.jfenn.attribouter.provider.net.ProviderString

open class UserData {
    @Transient
    open var id: ProviderString? = null
    open val login: String? = null
    open val name: String? = null
    open val avatarUrl: String? = null
    open val websiteUrl: String? = null
    open val email: String? = null
    open val bio: String? = null
}